### DISCUSSION TITLE
Quels sont les différents types de personnalité du MBTI ? Comment s'en servir ?

### DISCUSSION THREAD

**Discussion by christianfelicite** [TheFeloDevTeam/FeloFamilySite#726] [Questions de philosophie]

**Comment by christianfelicite** [comment]
https://nospensees.fr/8-types-de-personnalite-selon-carl-jung/

# Caractère de type extraverti

Caractéristiques du caractère extraverti :
- L’intérêt est focalisé sur la réalité extérieure avant le monde interne.
- Les décisions sont prises en fonction de l’effet produit sur la réalité externe plutôt que sur l’existence propre.
- Les actions sont réalisées en fonction de ce que les autres peuvent penser.
- L’éthique et la morale se construisent selon ce qui prédomine dans le monde.
- Ces personnes sont pratiquement à l’aise dans tous les environnements, mais il leur est difficile de s’adapter réellement.
- Elles sont suggestibles, influençables et ont tendance à imiter.
- Elles ont besoin de se faire voir et d’être reconnues par les autres.

# Caractère de type introverti

Caractéristiques du caractère introverti :
- L’intérêt est focalisé sur elles-mêmes, leurs sentiments et leurs pensées.
- Elles orientent leur conduite en fonction de ce qu’elles ressentent et ce qu’elles pensent, même si cela va à l’inverse de la réalité extérieure.
- Elles ne se préoccupent pas de l’effet de leurs actions sur leur environnement. Elles cherchent surtout à être satisfaites intérieurement.
- Elles ont des difficultés à se sentir à l’aise mais aussi à s’adapter à l’environnement. Cependant, si elles parviennent à s’adapter, elles le feront réellement et de manière créative.

**Comment by christianfelicite** [comment]
https://nospensees.fr/8-types-de-personnalite-selon-carl-jung/

Les types de personnalité

Selon Jung, à partir des fonctions psychologiques de base et des deux types de caractères fondamentaux, il existe huit types de personnalité bien distinctes.

Chacun de nous appartiendrait à l’un de ces groupes. Les voici :

# Réfléchie extravertie

La personnalité réfléchie extravertie correspond aux individus cérébraux et objectifs, qui agissent quasiment exclusivement en fonction de la raison.

Ils ne prennent pour acquis que ce qui est affirmé avec des preuves suffisantes. Ces personnes sont peu sensibles et peuvent même parfois être tyranniques et manipulatrices avec les autres.

# Réfléchie introvertie

Le réfléchi introverti est une personne avec une grande activité intellectuelle, mais qui a du mal à se lier aux autres.

C’est une personne généralement obstinée et très tenace au moment d’atteindre ses objectifs. Parfois, on la voit inoffensive et intéressante à la fois.

# Sentimentale extravertie

Les sentimentaux extravertis sont des personnes ayant une grande capacité à comprendre les autres et à établir des relations sociales.

Cependant, il leur est difficile de s’éloigner du troupeau et elles souffrent lorsqu’elles sont ignorées par leur entourage. Enfin, ces personnes sont très adroites en matière de communication.

# Sentimentale introvertie

La personnalité sentimentale introvertie correspond aux personnes solitaires ayant beaucoup de mal à établir des relations avec les autres.

Elles peuvent être maussades et mélancoliques. Elles font tout leur possible pour passer inaperçues et elles aiment rester dans le silence.

Cependant, elles sont très sensibles aux besoins des autres.
Lo que no se ve tras la máscara (personalidad oculta)

# Perceptive extravertie

Les individus perceptifs extravertis ont une faiblesse particulière pour les objets, auxquels ils attribuent même des qualités magiques, même s’ils le font de façon inconsciente.

Les idées ne les passionnent pas, contrairement aux corps concrets. Ils cherchent le plaisir avant tout.

# Perceptive introvertie

Ce type de personnalité concerne bien souvent les musiciens et les artistes.

En effet, les personnes perceptives introverties se focalisent tout particulièrement sur les expériences sensorielles: elles accordent une grande valeur à la couleur, à la forme, à la texture…

Elles aiment le monde de la forme et les expériences intérieures.

# Intuitive extravertie

C’est la personnalité type de l’aventurier. Les personnes intuitives extraverties sont très actives et inquiètes. Elles ont énormément besoin de stimulation en tout genre.

Elles sont tenaces pour atteindre leurs objectifs et une fois qu’elles y sont parvenues, elles passent au suivant en oubliant le précédant.

Le bien-être des personnes qui les entourent importe peu.

# Intuitive introvertie

Ces personnes sont extrêmement sensibles aux stimulations les plus subtiles. La personnalité intuitive introvertie correspond aux personnes qui « devinent » pratiquement ce que les autres pensent, ressentent ou s’apprêtent à faire.

Elles sont imaginatives, rêveuses et idéalistes. Cependant, il leur est difficile de « garder les pieds sur terre ».

**Comment by christianfelicite** [comment]
https://www.espritsciencemetaphysiques.com/il-existe-16-grands-types-de-personnalite.html

Il suffit de repérer ce qui est pour vous le plus spontané au cours de 4 « processus mentaux » suivants :
- Où puisez-vous votre énergie : dans votre univers intérieur (introversion « I »), ou à partir de l’environnement extérieur (extraversion « E ») ?
- Comment recueillez-vous l’information : par vos 5 sens (la sensation « S »), ou en vous confiant à votre « 6ème sens »  (l’intuition « N ») ?
- Qu’est-ce qui entraîne votre décision : le raisonnement logique (la pensée / think ou « T »), ou vos valeurs (sentiment/ feel ou « F ») ?
- De quelle manière vous lancez-vous dans l’action : en échafaudant des plans (jugement « J »), ou en vous adaptant aux circonstances (perception « P ») ?

http://www.klap.io/mbti-gerer-atelier-de-design-thinking/

![Prefs(2).png](https://user-images.githubusercontent.com/24352234/90331402-3dabd480-dfb4-11ea-93ac-800dd1fae499.png)

**Comment by christianfelicite** [comment]
https://fr.m.wikipedia.org/wiki/ISTP_(type_de_personnalit%C3%A9)

I – Introversion, préférée à l'extraversion : les ISTP sont généralement discrets et réservés. Ils préfèrent interagir avec quelques amis dont ils sont proches qu'avec un large cercle de connaissances, et ils dépensent (c'est-à-dire perdent) de l'énergie lors de leurs relations en société (tandis que les types extravertis, dans les mêmes circonstances, en gagnent)[3].

S - Sensation, préférée à l'intuition : les ISTP raisonnent davantage par le concret que par l'abstrait. Ils concentrent leur attention sur les détails plutôt que sur une vision globale des choses, et sur les réalités immédiates plutôt que sur les possibilités futures[4].

T – Pensée (Thinking), préférée au sentiment : les ISTP placent les critères objectifs au-dessus des préférences personnelles. Lorsqu'ils prennent une décision, ils accordent une importance plus grande à la logique qu'à des considérations sociales et/ou passionnelles[5].

P - Perception préférée au jugement : les ISTP retiennent leur jugement et mettent du temps à prendre des décisions importantes, préférant garder un maximum de possibilités ouvertes au cas où les circonstances changeraient.

**Comment by christianfelicite** [comment]
https://fr.m.wikipedia.org/wiki/ISFP

# I – Introversion, préférée à l'extroversion

Les ISFP sont généralement discrets et réservés. Ils préfèrent interagir avec quelques amis dont ils sont proches qu'avec un large cercle de connaissances, et ils dépensent (c'est-à-dire perdent) de l'énergie lors de leurs relations en société (tandis que les types extravertis, dans les mêmes circonstances, en gagnent)[3].
    
# S - Sensation, préférée à l'intuition

Les ISFP raisonnent davantage par le concret que par l'abstrait. Ils concentrent leur attention sur les détails plutôt que sur une vision globale des choses, et sur les réalités immédiates plutôt que sur les possibilités futures[4].
 
# F - Sentiment (Feeling) préféré à la pensée

Les ISFP valorisent davantage les considérations subjectives ou personnelles que les critères impersonnels et objectifs. Lorsqu'ils prennent des décisions, ils accordent un poids plus grand à des considérations sociales qu'à la logique[5].
    
# P - Perception préférée au jugement

Les ISFP retiennent leur jugement et mettent du temps à prendre des décisions importantes, préférant garder un maximum de possibilités ouvertes au cas où les circonstances changeraient

**Comment by christianfelicite** [comment]
http://www.sobienetre.fr/test-16-personnalites-mbti-arnaque/

**Comment by christianfelicite** [comment]
https://fr.m.wikipedia.org/wiki/Myers_Briggs_Type_Indicator

**Comment by christianfelicite** [comment]
http://www.16-types.fr/16types.html

**Comment by christianfelicite** [comment]
https://personalityjunkie.com/entp-a-vs-entp-t-personality-type/

**Comment by christianfelicite** [comment]
https://www.16personalities.com/articles/identity-assertive-vs-turbulent

Identity: Assertive vs. Turbulent

# Assertive (-A) Personality => Calm in the Storm

93% of those with the Assertive Identity say they feel confident facing day-to-day difficulties, compared to 62% of those with the Turbulent Identity.

# Turbulent (-T) Personality => Power of Stress, Activate!

Turbulent individuals are success-driven, perfectionistic, and eager to improve.

They are always trying to counterbalance their self-doubts by achieving more.

Just as they push themselves to become better, they’re as likely to push their projects or efforts in the same direction. Turbulent personality types tend to notice little problems and often do something about them before they become larger ones.

86% of those with the Turbulent Identity say that comparing themselves to others usually results in a negative feeling, compared to 58% of those with the Assertive Identity.

**Comment by christianfelicite** [comment]
https://www.16personalities.com/articles/the-turbulent-personality-dont-worry-be-happy

**Comment by christianfelicite** [comment]
https://www.16personalities.com/fr/test-de-personnalite

https://www.16personalities.com/fr/types-de-personnalite

# Les diplomates
 
## Advocate (INFJ) Personality - Avocat
INFJ-A / INFJ-T
Idéalistes calmes et mystiques et pourtant très inspirants et infatigables.

## Mediator (INFP) Personality - Médiateur
INFP-A / INFP-T
Personnes poétiques, gentilles et altruistes qui sont toujours prêtes à soutenir une bonne cause.

## Protagonist (ENFJ) Personality - Protagoniste
ENFJ-A / ENFJ-T
Leaders charismatiques et inspirants, capables de fasciner leur public.

## Campaigner (ENFP) Personality - Inspirateur
ENFP-A / ENFP-T
Esprits libres enthousiastes, créatifs et sociables, qui arrivent toujours à trouver une raison de sourire.

**Comment by christianfelicite** [comment]
https://www.16personalities.com/fr/test-de-personnalite

https://www.16personalities.com/fr/types-de-personnalite

# Les sentinelles

## Logistician (ISTJ) Personality - Logisticien
ISTJ-A / ISTJ-T
Individus pragmatiques et intéressés par les faits, dont le sérieux ne saurait être mis en cause.

## Defender (ISFJ) Personality - Défenseur
ISFJ-A / ISFJ-T
Protecteurs très dévoués et très chaleureux, toujours prêts à défendre ceux qu’ils aiment.

## Executive (ESTJ) Personality - Directeur
ESTJ-A / ESTJ-T
Excellents gestionnaires, d’une efficacité inégalée quand il s’agit de gérer les choses, ou les gens.

## Consul (ESFJ) Personality - Consul
ESFJ-A / ESFJ-T
Personnes extraordinairement attentionnées, sociables et populaires, toujours prêtes à aider les autres.

**Comment by christianfelicite** [comment]
https://www.16personalities.com/fr/test-de-personnalite

https://www.16personalities.com/fr/types-de-personnalite

# Les explorateurs

## Virtuoso (ISTP) Personality - Virtuose
ISTP-A / ISTP-T
Expérimentateurs hardis et pragmatiques, maîtres de toutes sortes d’outils.

## Adventurer (ISFP) Personality - Aventurier
ISFP-A / ISFP-T
Artistes flexibles et charmants, toujours prêts à explorer et à essayer quelque chose de nouveau.

## Entrepreneur (ESTP) Personality - Entrepreneur
ESTP-A / ESTP-T
Personnes astucieuses, énergiques et très perspicaces, qui aiment vraiment vivre à la pointe du progrès.

## Entertainer (ESFP) Personality - Amuseur
ESFP-A / ESFP-T
Amuseurs spontanés, énergiques et enthousiastes; avec eux, on ne s’ennuie jamais.

**Comment by christianfelicite** [comment]
https://www.16personalities.com/fr/test-de-personnalite

https://www.16personalities.com/fr/types-de-personnalite

# Les analystes

## Architect (INTJ) Personality - Architecte
INTJ-A / INTJ-T
Penseurs imaginatifs et stratèges, avec un plan pour tout.

## Logician (INTP) Personality - Logicien
INTP-A / INTP-T
Inventeurs innovateurs dotés d’une soif inextinguible de connaissances.

## Commander (ENTJ) Personality - Commandant
ENTJ-A / ENTJ-T
Leaders hardis, imaginatifs et dotés d’un fort caractère, qui trouvent toujours un moyen d’arriver à leurs fins, ou le créent.

## Debater (ENTP) Personality - Innovateur
ENTP-A / ENTP-T
Penseurs astucieux et curieux incapables de résister à un défi intellectuel.

**Comment by christianfelicite** [comment]
http://www.klap.io/mbti-gerer-atelier-de-design-thinking/

1/ Pour le Facilitateur : mieux se connaître pour adopter une meilleure posture

Pour le facilitateur, le MBTI est un outil de connaissance et de développement personnel.

La 1ère chose à faire pour le facilitateur est de connaître son profil psychologique. Et pour cela rien de plus simple : il faut le faire avec un coach certifié MBTI pour avoir une étude précise et certifiée – Plus d’infos ici.

La 2ème chose à faire, c’est d’utiliser cette information. Connaître son profil, c’est bien, apprendre sur soi et comprendre comment on fonctionne, c’est encore mieux 😉

Cela permet d’identifier ses atouts, caractéristiques propres, la façon dont les autres nous perçoivent et ses zones de développement personnel.

Utiliser ces infos pour développer ses compétences en tant que Facilitateur est bien utile pour connaître ses zones de confort et d’effort, savoir comment on se connecte aux autres.

Et répondre à toute une série de questions liées à ces préférences :

    La connexion aux autres est-elle génératrice d’énergie ou pas (suivant sa préférence E ou I) ?
    Ma façon de communiquer est-elle « big picture » ou dans le détail (préférence N ou S) ?
    Ma façon de prendre des décisions rationnelle ou empathique (préférence T vs F) sachant que je dois rester neutre par rapport au groupe et par rapport aux décisions du groupe
    Comment je vais organiser mon atelier : en anticipant et en cadrant au maximum (préférence J), ou plus dans l’agilité (préférence P) ?
    Comment je vais agir avec d’autres facilitateurs qui n’ont pas le même profil que moi : une personne de Préférence P (structuré fluide) vs Préférence J (structuré organisé) ?
    Comprendre aussi son évolution avec l’âge et comment parfaire son profil, vers quelles zones d’effort aller pour continuer de se développer ?

Ce ne sont ici que quelques premières questions, mais on entrevoit aisément le potentiel de ces connaissances sur sa posture

**Comment by christianfelicite** [comment]
http://www.klap.io/mbti-gerer-atelier-de-design-thinking/

2/ Pour les groupes : faire des combinaisons de profils qui ont du sens pour une intelligence collective amplifiée

En intelligence collective, la diversité des profils et des points de vue crée la richesse.

Connaître les profils va permettre de rajouter une dimension pour constituer les groupes (en plus d’autres critères comme le sexe, la profession, leur fonction…).

Attention à bien intégrer que certains groupes ont des spécificités liées à leur profession : on trouve beaucoup de profils commençant par ENF chez les facilitateurs, de combinaison des préférences NF chez les coaches ; de profils ISTJ chez les les Ingénieurs, de combinaisons NT pour les Marketeurs… Il faudra ainsi en tenir compte et favoriser la diversité (autant que possible) des profils.

Connaissant les spécificités d’un groupe, on pourra ainsi anticiper les apports ou les points de vigilance des participants pour le groupe, et essayer de mixer au maximum les préférences.

Par exemple, on sait grâce au MBTI que les personnes de préférence E apportent dans un groupe dynamisme, facilité de contacts, goût de l’action, et circulation d’énergie et que leurs points de vigilance seront leur excès de parole, leur manque d’écoute, leur action irréfléchie ou leurs remarques blessantes. A charge au Facilitateur de prendre ces éléments en compte pour chaque dimension pour gérer au mieux les personnes et les groupes.

**Comment by christianfelicite** [comment]
http://www.klap.io/mbti-gerer-atelier-de-design-thinking/
3/ Pour l’atelier : se connecter aux autres devient plus simple

Tout au long d’une démarche de type Design Thinking, les informations liées aux préférences seront bien utiles.

En amont et en aval d’un atelier : que ce soit avec le commanditaire ou avec l’équipe qui mène la démarche, on a une meilleure compréhension du contexte, des intérêts cachés, des pièges à éviter.

Pour le travail avec le Commanditaire par exemple, c’est souvent utile au delà de l’aspect technique, d’anticiper certains besoins, réactions, et de voir comment on se situe par rapport à lui, afin de mieux comprendre et répondre à ses attentes.

Lors des ateliers, et notamment lors d’un atelier de Design Thinking, c’est souvent mieux pour comprendre les interactions lors des différentes phases :

    Lors de la Phase Immersion, où les préférences E/ S/ F vont être particulièrement sollicitées
    Pendant la Phase Ideation, où la part belle sera aux personnes de préférences E/ N/ T/ P)
    Pour la Phase de Prototypage, où pourront s’illustrer les personnes de préférence S pour la qualité des finitions
    Et enfin pour la Phase de Test (préférences E/ T pour le déroulé/ préférence J pour la structure du pitch…)

Et aussi, dans les différents exercices collectifs, quand on a besoin de diverger : les personnes de préférence N seront bien utiles / pour converger : celles de préférences T / pour donner du sens : préférence F / pour sortir du cadre : préférence P / pour rendre les livrables dans les temps : préférence J

Ce sont ici quelques pistes de réflexion, et c’est avec l’expérience que sera développée la compétence, réel atout pour le Facilitateur.

**Comment by christianfelicite** [comment]
http://le-mbti.e-monsite.com/articles/article-f5-qualites-et-defauts-des-differents-types-de-personnalite.html

**Comment by christianfelicite** [comment]
https://www.penserchanger.com/mbti-transformer-votre-vie/amp

**Comment by christianfelicite** [comment]
https://intelligence-performancielle.com/le-mbti-autrement-tu-utiliseras-3/

Et si l’application première du MBTI était d’améliorer ses relations ?

En cinq étapes, une meilleure relation tu développeras

Notre proposition se veut simple et facilement applicable au quotidien.

    
# Écouter tu feras.

Il s’agit d’abord d’écouter, non seulement le contenu, mais surtout les préférences qui sont en train de s’exprimer dans le discours ou le comportement de l’autre. Á titre d’exemple, je suis en réunion et l’un de mes interlocuteurs parle beaucoup alors que j’ai l’impression qu’il ferait mieux de se taire pour réfléchir avant de parler. Cette situation me gêne.

    
# Les préférences tu identifieras.

Quelle préférence suis-je en train d’identifier chez chacun ? Il n’est pas nécessaire de maîtriser le modèle complet du MBTI. Il suffit d’envisager la situation en termes de bénéfices. Quel bénéfice peut avoir mon interlocuteur à prendre la parole alors que ses idées ne semblent pas encore claires de mon point de vue ? Je suis alors capable de faire une hypothèse, qui sera validée par la suite, ou pas. Par exemple, je peux envisager qu’il cherche à formaliser et clarifier son point de vue en verbalisant immédiatement toutes ses idées, même si elles sont encore désorganisées. Et ça, c’est un bénéfice pour lui ! Et je fais l’hypothèse qu’il est de préférence extraversion.

    
# Tes difficultés dans l’instant et dans ta tête tu formaliseras.

Dans cette situation, ma difficulté est par exemple de ne pas réussir à avoir moi-même les idées claires par cette profusion d’idées trop larges et mal canalisées. Je peux également avoir l’impression que nous n’arriverons pas à mettre en place le plan d’actions attendu par notre hiérarchie dans les délais prévus. Ce qui est essentiel ici, c’est de réaliser et d’accepter que toutes ces raisons viennent de mon propre ressenti impression du moment face à la situation vécue.

    
# D’en parler éventuellement tu décideras.

Il est parfois difficile d’envisager d’en parler avec l’autre, tant que je n’ai pas réussi à mettre en évidence les bonnes raisons et les bénéfices qui dictent ses comportements. Je vais alors avoir tendance à penser qu’il ne peut pas comprendre, et qu’il vaut mieux ne rien dire. Ne pas en parler revient à garder un œil accusateur sur l’autre et à préserver sa propre frustration. Mais, si vous en parliez en mettant en avant le bénéfice supposé de votre interlocuteur, que se passerait-il ? Je pourrais dire par exemple : j’ai l’impression que c’est utile pour toi de verbaliser toutes tes idées pour les clarifier, mais en même temps, je suis inquiet quant au respect du délai qui nous est imparti, qu’est-ce que tu en penses et est-ce que tu crois qu’on va y arriver ?

    
# Une posture adaptée tu envisageras.

Si votre interlocuteur a des difficultés pour fonctionner différemment, il va bien falloir que la personne la plus à même de le faire l’envisage. Dans le cas présent, peut-être vous faudra-t-il l’aider à formaliser ses pensées en les reformulant avec lui, dans son mode de préférence extraversion. Dans bien des cas, la situation s’améliore spontanément par le fait même d’avoir énoncé la problématique en termes de bénéfices pour chacun. Dans les faits, chacun va essayer d’aller vers la préférence de l’autre, par exemple : la personne ayant une préférence extraversion va essayer d’être plus synthétique et la personne ayant une préférence introversion acceptera le temps nécessaire aux reformulations plus courtes. C’est ce qui permet d’amoindrir les zones d’inconfort et résorber les exaspérations et les sources de conflit.

**Comment by verasqlbtzdti770-lgtm** [comment]
Votre approche consiste surtout à observer le bénéfice recherché derrière un comportement, puis à formuler une hypothèse sur la préférence MBTI, sans la considérer comme une certitude. Verbaliser des idées encore désordonnées peut aider certaines personnes à les clarifier. L’hypothèse se vérifie ensuite. Et pour changer de terrain, [plus d'infos ici](https://fatbossconnexion.fr/) peut offrir une pause ludique après cette réflexion.
